src/posthog-core.ts

```diff
@@ -1026,6 +1032,26 @@ export class PostHog {
      * between IDs at this time, so when you change a user's ID
      * they will appear to be a new user.
      *
+     * Users are marked as 'identified' when an $identify event is sent to posthog.
+     * See https://github.com/PostHog/posthog-js/issues/512 for more context.
+     *
+     * This means if (in a single browser "session") you call
+     *
+     * identify('a') => capture('event 1') => identify('b') => capture('event 2')
+     *
+     * event 1 is linked to person a, and event 2 is linked to person b.
+     * person a and person b are not merged
+     *
+     * Users are marked as 'anonymous' when reset is called and in some cases on init.
+     *
+     * This means if (in a single browser "session") you call
+     *
+     * identify('a') => capture('event 1') => reset() => capture('event 2') => identify('b') => capture('event 3')
+     *
+     * person a and person b are merged, and event 1, 2, and event 3 are linked to person b.
+     *
+     * If you need more control over how users are merged, you can use the alias method.
```

Comment:
you mean because of the hackery we do in posthog-js, as in ingestion they are exactly the same